src/styles/_catalog.scss

```diff
@@ -1,22 +1,39 @@
 .catalog {
-  display: flex;
+  display: grid;
   flex-wrap: wrap;
 
   max-width: 944px;
   margin: 0 auto;
 
+  grid-template-columns: repeat(auto-fill, $cardWidth);
+  gap: 48px;
+  justify-content: center;
+
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $cardWidth;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media (max-width: 1024px) {
```

Comment:
do you need `@media` since you are using `auto-fill`? you can just add correct padding to `.catalog`